src/test/ui/borrowck/two-phase-across-loop.stderr

```diff
@@ -2,7 +2,21 @@ error[E0499]: cannot borrow `foo` as mutable more than once at a time
   --> $DIR/two-phase-across-loop.rs:17:22
    |
 LL |         strings.push(foo.get_string());
-   |                      ^^^^^^^^^^^^^^^^ `foo` was mutably borrowed here in the previous iteration of the loop
+   |         -------------^^^^^^^^^^^^^^^^-
+   |         |            |
+   |         |            `foo` was mutably borrowed here in the previous iteration of the loop
+   |         first borrow later used here
+   |
+help: try adding a local storing this...
+  --> $DIR/two-phase-across-loop.rs:17:22
+   |
+LL |         strings.push(foo.get_string());
+   |                      ^^^^^^^^^^^^^^^^
+help: ...and then using that local here
+  --> $DIR/two-phase-across-loop.rs:17:9
+   |
+LL |         strings.push(foo.get_string());
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Not quite understand this help message, seems a little bit confusing, it's suggesting to fix code like this?

```rust
 let x = foo.get_string().to_string();
 strings.push(x); 
```

But we can fix it like this:
```rust
 strings.push(foo.get_string().to_string()); 
```

Reply:
Yes, this is bit confusing. Since this is a generic suggestion, not fit for this case. I have taken it out for the case.